theme-contract/src/main/java/com/orange/ouds/theme/OudsAdaptiveDisplay.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Software Name: OUDS Android
+ * SPDX-FileCopyrightText: Copyright (c) Orange SA
+ * SPDX-License-Identifier: MIT
+ *
+ * This software is distributed under the MIT license,
+ * the text of which is available at https://opensource.org/license/MIT/
+ * or see the "LICENSE" file for more details.
+ *
+ * Software description: Android library of reusable graphical components
+ */
+
+package com.orange.ouds.theme
+
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+
+/**
+ * A class to represent the adaptive displays managed in OUDS.
+ */
+enum class OudsAdaptiveDisplayType {
+    EXTRA_COMPACT, COMPACT, MEDIUM;
+
+    companion object {
+        /**
+         * Returns the [OudsAdaptiveDisplayType] for the given [windowWidth] in dp.
+         */
+        fun fromWindowWidth(windowWidth: Dp): OudsAdaptiveDisplayType {
```

Comment:
I wonder if we should add an equivalent of `currentWindowAdaptiveInfo()` in order to avoid writing the following line each time we want to call this method:

```kotlin
val windowWidth = with(LocalDensity.current) { currentWindowSize().toSize().toDpSize().width }
```

Reply:
I think we could rename our `currentWindowAdaptiveInfo` method to `currentWindowWidth` because it returns a width in `Dp` instead of a `WindowAdaptiveInfo` in the Material equivalent. Moreover, it would avoid confusion between the two methods.